saleor/payment/gateways/np_atobarai/api_helpers.py

```diff
@@ -165,29 +165,49 @@ def _get_voucher_and_shipping_goods(
     return goods_lines
 
 
-def get_refunded_goods(
+def get_goods_with_refunds(
     config: "ApiConfig",
-    refund_data: RefundData,
     payment_information: PaymentData,
 ) -> List[dict]:
-    """Convert payment lines and refund lines into NP Atobarai goods lines.
+    goods_lines = []
```

Comment:
This method previously had a docstring. Why you didn't add it now?

Reply:
Resolved in recent commit.